src/tutorials/0009-introduction-to-libp2p/01.md

```diff
@@ -7,47 +7,43 @@
 
-## Location Addressing vs Content Addressing
+There are so many things that can go wrong in this scenario, and the truth is, you probably didn't want the photo to be stored in some far away database which you can't delete. You just wanted to share your photo directly with your friend.
 
-A lot of the issues that we face, come from using a [Location Addressing](/data-structures/02) approach to find the content the user is looking for, and serve it from that address. For example, we expect that our `libp2p` logo to be available at [https://proto.school/tutorial-assets/T0009L01-libp2p-logo.svg](/tutorial-assets/T0009L01-libp2p-logo.svg), but what if the server of this website is down? What if the DNS server is down? What if the image was converted to a PNG file and so the new location is `T0009L01-libp2p-logo.png`? What if you are in a country that has blocked the `proto.school` domain?
-All of these issues are common ones and are caused by `Location Addressing`, where we try to find the content we need by it's location.
+## Location Addressing vs Content Addressing
 
-[IPFS](https://ipfs.io), however, approaches this problem in a different way by using [Content Addressing](/data-structures/03) instead.
-Before, we knew the location of our libp2p logo image, but, if instead we know it's [CID](/anatomy-of-a-cid) (Content IDentifier) we can then ask who has the image with this CID.
-The advantages in this approach are very appealing since they solve all the other issues before mentioned, but to get it right there are other problems that need to be tackled first, specifically within networking.
+A lot of the issues that we face come from using a [location addressing](/content-addressing/02) approach to find the content the user is looking for, and serving it from that address. For example, we expect our libp2p logo to be available at [https://proto.school/tutorial-assets/T0009L01-libp2p-logo.svg](/tutorial-assets/T0009L01-libp2p-logo.svg), but what if the server of this website is down? What if the DNS server is down? What if the image was converted to a PNG file and so the new location is `T0009L01-libp2p-logo.png`? What if you are in a country that has blocked the `proto.school` domain?
+All of these issues are common ones and are caused by location addressing, where we try to find the content we need by its location.
 
+[IPFS](https://ipfs.io), however, approaches this problem in a different way by using [content addressing](/content-addressing/03) instead. If we know the
+[CID](/anatomy-of-a-cid) (Content IDentifier) of our libp2p logo instaed of its location, we can simply ask the network who has the image with this CID.
```

Comment:
```suggestion
[IPFS](https://ipfs.io), however, approaches this problem in a different way by using [content addressing](/content-addressing/03) instead. If we know the [CID](/anatomy-of-a-cid) (Content IDentifier) of our libp2p logo instead of its location, we can simply ask the network who has the image with this CID.
```